packages/plugin-testing-integration/src/app.js

```diff
@@ -151,8 +145,48 @@ async function initImaApp(bootConfigMethods = {}) {
     };
   }
 
-  let app = createImaApp();
-  let bootConfig = getClientBootConfig({
+  /**
+   *Get response content for the application run
+   *
+   * @returns {string} html content
+   */
+  async function _getIMAResponseContent() {
+    const devUtilsFactory = jest.mock(
```

Comment:
This adds dependency to `jest`, but `jest` might not be available always, maybe this is forgotten code used just for testing purposes and needs to be rewritten to use some native ima server functions?